include/dlaf/factorization/qr/t_factor_mc.h

```diff
@@ -205,15 +211,18 @@ void QR_Tfactor<Backend::MC, Device::CPU, T>::call(const SizeType k, Matrix<cons
   // 1st step: compute the column partial result `t`
   // First we compute the matrix vector multiplication for each column
   // -tau(j) . V(j:, 0:j)* . V(j:, j)
-  for (const auto& v_i : iterate_range2d(v_start, v_end)) {
-    const bool is_v0 = (v_i.row() == v_start.row());
+  for (const auto& v_i : panel_view.iteratorLocal()) {
+    // TODO improve panel_view API (begin)
+    const bool is_v0 = (v_i.row() == panel_view.begin().row());
```

Comment:
This is wrong in case the diagonal falls across two different tiles.

Reply:
It can be fixed when working on the back-transformation.